src/handlers/authentication.ts

```diff
@@ -61,6 +61,23 @@ export async function authenticateWithPINAndCreateIfNeeded() {
   });
 }
 
+export async function saveNewAuthenticatePIN() {
```

Comment:
This method name is confusing for me. Did you mean `saveNewAuthenticationPIN`?

Reply:
Yes, I will rename that